The blue team column hides in the match breakdown when viewing remote events

khari05 opened the pull request:
the latest build of the app's match breakdown completely breaks in remote events. It now just hides the blue team column completely instead

khari05 committed:
the blue team column now hides in the match breakdown when viewing remote events

ofekashery commented on lib/ui/views/match/years/match-breakdown-2021.dart:
```diff
@@ -6,122 +6,144 @@ import '../../../widgets/match-breakdown-row.dart';
-  static List<Widget> getRows(Match match, BuildContext context) {
+  static List<Widget> getRows(Match match, BuildContext context, bool isRemote) {
     TOALocalizations local = TOALocalizations.of(context);
     UltimateGoalMatchDetails details = match.gameData;
```
It makes more sense to check how many teams participating in the match.
```suggestion
    UltimateGoalMatchDetails details = match.gameData;
    bool isRemote =  match.participants.length == 1;
```

ofekashery commented:
For me it seems more readable to hide the left column instead of the right (and reverse on RTL), also on the website.
cc @Techno11

khari05 commented:
If we changed the side, maybe it might be more maintainable if we made a different match breakdown page for remote events and made a `RemoteMatchBreakdownRow` since things like alliance color aren't really necessary in remote events.

nholo1332 commented:
Agreed, creating a new component might be best for maintainability. That way we aren't directly making it season-specific and allowing for the possibility of remote events in the future

khari05 committed:
breakdown is now separated into 2 files
a new remote match breakdown and a remote match breakdown row
the match list item now shows teams in remote events

khari05 commented:
a `RemoteMatchBreakdown2021` and `RemoteMatchBreakdownRow` renders remote events now
![image](https://user-images.githubusercontent.com/35894769/100551356-e0c7e780-324d-11eb-9891-6c073ba7b64f.png)

khari05 committed:
wobble goal end position displays correctly now

nholo1332 commented:
@khari05 Looks good in my opinion. Just to make the visuals less ambiguous, would it be best to make the background for one alliance orange instead of the current blue (and add some branding)?

khari05 commented:
I just left it as blue since I didn't know what color to set it to. Do you know which shades of orange would be equivalent to `lightBlue` and `lighterBlue`?

nholo1332 commented:
Try importing `import './ui/colors.dart' as TOAColors;` and using `TOAColors.Colors().toaColors.shade300` or whatever shade looks best. Let me know if that helps.

khari05 commented:
This image is using 300 and 200 for light mode while its using 700 and 500 for dark mode. How does it look?
![matchbreakdown](https://user-images.githubusercontent.com/35894769/100695844-19f07c80-3360-11eb-8ce7-01ecfe605d43.png)

nholo1332 commented:
I think that looks good

nholo1332 commented:
I think that looks good for the light mode! For the dark mode, maybe adding some transparency to the color would help? I'm not sure. If you can't find anything that works for that, we could leave dark mode with a darker or lighter color black (like on the score name area). In fact, the darker/lighter black might fit better with the dark mode theme in general. The only other thing that I can think of is changing the top color (currently light blue) to the light orange. And I think it's fine to use the `.withOpacity(0.5)` if the color you want isn't a shade.
Thank you for making all of these changes!

khari05 commented:
I couldn't get an orange that looks nice since when opacity is added, it looks a bit too washed out. Here's how it looks with a light black instead of orange.
I'll change the 
![Screen Shot 2020-12-01 at 3 17 17 PM](https://user-images.githubusercontent.com/35894769/100792376-929b1b80-33e8-11eb-88a1-bd2d21a67c66.png)

khari05 committed:
fixed coloring of the team column in the remote match breakdown

khari05 committed:
changed the color of teams in remote matches

khari05 committed:
sorting for remote matches sorts by team before match number
